variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantClassifier.java

```diff
@@ -33,20 +35,28 @@
      * @return Type of the variant
      */
     public static VariantType getVariantClassification(String reference, String alternate, int subSNPClass) {
-        String alphaRegEx = "[A-Z]+";
+        Pattern alphaRegExPattern = Pattern.compile("[A-Z]+");
 
         reference = reference.trim();
         alternate = alternate.trim();
 
-        if (!reference.equals("NOVARIATION")) {
+        boolean isRefAlpha = alphaRegExPattern.matcher(reference).matches();
+        boolean isAltAlpha = alphaRegExPattern.matcher(alternate).matches();
+
+        if (reference.equals("NOVARIATION")) {
+            if (subSNPClass == 6) {
+                return VariantType.NO_SEQUENCE_ALTERATION;
+            }
+        }
```

Comment:
I think that our code style is:
```
if {
   something;
} else { 
 something else;
}
```
instead of
```
if {
   something;
}
else {
   somethingElse;
}
```